federation/cmd/federation-apiserver/app/server.go

```diff
@@ -87,6 +87,8 @@ func Run(s *options.ServerRunOptions, stopCh <-chan struct{}) error {
 func NonBlockingRun(s *options.ServerRunOptions, stopCh <-chan struct{}) error {
 	// register all admission plugins
 	registerAllAdmissionPlugins(s.Admission.Plugins)
+	genericapiserver.RegisterAllAdmissionPlugins(s.Admission.Plugins)
```

Comment:
Having registration in one place is quite elegant. It is more explicit. On the other hand to figure out where generic plugins are registered one can always use the grep tool.